sample/QueueTrigger-FSharp/function.json

```diff
@@ -1,5 +1,4 @@
 {
-    "excluded": true,
```

Comment:
We were relying on these exclusions to keep the sample app responsive. Loading all the F# samples was making the sample app sluggish

Reply:
@mathewc, so it should be ok for dev(core) ?

Reply:
We definitely don't want to load them all. Either remove, move them to another folder or add the ones we want to the "functions" white list in host.json

Reply:
Moved to separate folder

Reply:
I think we should just remove these. If they're not being run or used, they won't be maintained. All the other sample functions are loaded/run regularly, and many have integration tests exercising them. I wouldn't be surprised if these F# samples are already out of date.